### ISSUE TITLE
`sblint` terminates without saying anything

### ISSUE THREAD

**Issue by jcguu95** [fukamachi/sblint#22]
```
$> sblint
$>
```

What does this mean? Does it mean that the tested system has nothing to be linted? With the verbose option, it seems that it just loads the system..

```
$ sblint -v
[INFO] Lint directory './'
[INFO] Lint file lispc.asd
[INFO] Lint system lispc.asd
[INFO] Loading a system: lispc
[INFO] Done
```

**Comment by cxxxr** [comment]
sblint calls sbcl's compile-file, asdf's load-system function, etc. and collects the warnings generated at that time.
If no warnings are raised, it is normal for nothing to remain in the log.
